oak/proto/storage_channel.proto

```diff
@@ -18,26 +18,33 @@ syntax = "proto3";
 
 package oak;
 
+import "oak/proto/policy.proto";
+
 // In all the request messages below, the storage_name field is used to partition data within an Oak
 // Module and is not exposed to the StorageProvider. The StorageChannel generates a unique storage
 // identifier from storage_name and the Oak Module identifier to prevent name collisions among Oak
 // Modules.
 
+message StorageDatum {
```

Comment:
Maybe `Item` (or `Entry`, or some variation) would be a more familiar term for developers?Vaguely inspired by https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem , which at some point I would like to investigate more (i.e. if we could emulate storage with that in a browser when running client-side... anyway, that's a separate discussion)

Reply:
Item SGTM.